### PR TITLE
Add wrapper: SUPPA

### PR DESCRIPTION
**Pull request by stephanflemming**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Github: https://github.com/comprna/SUPPA
Conda: https://anaconda.org/bioconda/suppa
Conda recipe: https://bioconda.github.io/recipes/suppa/README.html
(My) parameter overview: https://docs.google.com/spreadsheets/d/1v0X9-4M14Q_jueh7lYxMdtSMA0l3rTTOi_QeUZaXVY0/edit?usp=sharing

Fix https://github.com/galaxyproject/tools-iuc/issues/2849

### CODE REVIEW

**Comment by stephanflemming** [comment]
A first - more or less working - version.

Test data needs to be replaced and I couldn't test clusterevents and therefore disabled it for the moment.

ping @bgruening @teresa-m

File: tools/suppa/diffsplice.xml [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+**What it does**
+
+@WID@
+
+SUPPA reads the PSI for the (transcripts or local) events and the transcript expression values from multiple samples, grouped by condition, and the ioe/ioi file, to calucate the events that are differentially spliced between a pair of conditions. SUPPA can test multiple conditions with a variabe number of samples per condition.
+
+**Input**
+
+The ioe/ioi file, the PSI files and the transcript expression files are required.
```
Comment (bgruening) [inline tools/suppa/diffsplice.xml]: where do you get them?

File: tools/suppa/clusterevents.xml [hunk 2]
```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0"?>
+<tool id="suppa_clusterevents" name="SUPPA clusterEvents" version="@TOOL_VERSION@+@WRAPPER_VERSION@">
+    <description></description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <stdio></stdio>
+    <command detect_errors="exit_code"><![CDATA[
+        ## run
```
Comment (bgruening) [inline tools/suppa/clusterevents.xml]: this tool is doing nothing ...

Comment (stephanflemming) [reply tools/suppa/clusterevents.xml]: I know, my test data caused a lot of warnings, disabled the tool for the moment.

File: tools/suppa/diffsplice.xml (hunk 1 above)
Comment (stephanflemming) [reply tools/suppa/diffsplice.xml]: > @stephanflemming if those tools need to come strictly one after the other, can this not be one bigger tool?

Good idea.

**Comment by stephanflemming** [comment]
Some minor questions are still open, beside that, this PR can be reviewed: https://github.com/comprna/SUPPA/issues/103

GenerateEvents, psiPerIsoform, psiPerEvent are merged into one wrapper. diffSplice and 

suppa

**Comment by stephanflemming** [comment]
Some minor things are still not clear: https://github.com/comprna/SUPPA/issues/103

Beside this, PR can be reviewed.

suppa

**Comment by bgruening** [comment]
Awesome! Let's wait for the comments from upstream 1-2 days?

**Comment by bernt-matthias** [comment]
Seems that the upstream issue got an answer ... Guess the PR could need a rebase to make testing complete.

**Pull request closed without merging**